docs/topics/exceptions.rst

```diff
@@ -77,3 +78,25 @@ NotSupported
 
 This exception is raised to indicate an unsupported feature.
 
+StopDownload
+-------------
+
+.. versionadded:: 2.2
+
+.. exception:: StopDownload(errback=False)
```

Comment:
Sorry, one more query, about naming :) `errback` can be confused with errback argument of Request; previously all "errback" arguments were functions, not bools. I wonder if it is possible to have a flag name without this issue. Some ideas (more ideas are welcome):

* `StopDownload(fail=True)`, `StopDownload(fail=False)`
* `StopDownload(ok=True)`, `StopDownload(ok=False)`
* `StopDownload` and `CancelDownload`
* `StopDownload` and `IgnoreRequest`

Reply:
I like `StopDownload(fail)`. Thanks!